root/static/scripts/release/components/BatchCreateWorksDialog.js

```diff
@@ -182,8 +182,22 @@ const BatchCreateWorksDialogContent = React.memo<
     sourceDispatch,
   ]);
 
+  const handleKeyDown = React.useCallback((event) => {
```

Comment:
I think it'd make sense to add a custom hook for this, like `useDialogEnterKeyHandler` or something similarly named, which returns the `React.useCallback`.

Then you can just do `const handleKeyDown = useDialogEnterKeyHandler(acceptDialog);` in all three places.